coralproject/talk
# [CORL 547] org-wide announcements

## What does this PR do?
- adds config section to "General" pane allowing creation and deletion of announcements
- displays announcements on stream for a specified duration
- allows users to dismiss announcements

## How do I test this PR?
- navigate to configuration -> Community announcement
- create an announcement
- check that announcement is present above stream
- dismiss announcement from stream
- remove announcement from admin
- create a new announcement
- new announcement should display on stream
